decidim-core/app/views/decidim/reported_mailer/hidden_by_admin.html.erb

```diff
@@ -0,0 +1,25 @@
+<p class="email-greeting"><%= t(".hello", name: @user.name) %></p>
+
+<p class="email-instructions">
+  <%= t(".report_html", url: reported_content_url, admin: @admin.name) %>
```

Comment:
```suggestion
  <%= t(".report_html", url: reported_content_url, moderator: @moderator.name) %>
```

Reply:
Suggestion applied in 8555f7f9dd . Done without Github's help as we needed to rename a file.